R/manifest.R

```diff
@@ -221,7 +221,26 @@ fill_dataflow_manifest <- function(dataflow_manifest_chunk,
       all.x = TRUE
     )
   }
-
+  
+  # If modified_on is an attribute, get the modified_on of each manifest
+  if ("modified_on" %in% attributes_df$Attribute) {
+    files <- lapply(dataflow_manifest_chunk$dataset_id, function(x) {
+      entities <- dfamodules::synapse_entity_children(auth = access_token,
+        parentId = x, includeTypes = list("file"))
+      entities$dataset_id <- x
+      entities
+    })
+    files <- dplyr::bind_rows(files)
+    dplyr::filter(files, grepl("synapse_storage_manifest", name))
+    files <- dplyr::select(files, modified_on=modifiedOn, dataset_id)
+    dataflow_manifest_chunk <- dplyr::select(dataflow_manifest_chunk, -modified_on)
```

Comment:
Need to test this, I am not sure if it's having the desired effect. I put it in to avoid merge results with modified_on.x and modified_on.y